src/ios/IdfaPlugin.m

```diff
@@ -40,11 +22,26 @@ - (void)getInfo:(CDVInvokedUrlCommand *)command {
 
         CDVPluginResult* pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsDictionary:@{
             @"idfa": idfaString,
-            @"limitAdTracking": [NSNumber numberWithBool:!enabled],
-            @"trackingTransparencyStatus": trackingTransparencyStatus
+            @"trackingLimited": [NSNumber numberWithBool:!enabled],
+            @"trackingPermission": trackingPermission ?: [NSNull null]
         }];
         [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
     }];
 }
 
+- (void)requestPermission:(CDVInvokedUrlCommand *)command {
+    [self.commandDelegate runInBackground:^{
+        if (@available(iOS 14, *)) {
+            [ATTrackingManager requestTrackingAuthorizationWithCompletionHandler:^(ATTrackingManagerAuthorizationStatus status) {
+                CDVPluginResult* pluginResult =
+                    [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsNSUInteger:status];
+                [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
+            }];
+        } else {
+            CDVPluginResult* pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK];
+            [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
```

Comment:
Lets send an error response here with message "requestPermission is supported only for iOS >= 14"